### DISCUSSION TITLE
Free energy calculations of competing CDW phases

### DISCUSSION THREAD

**Discussion by kw600** [SSCHAcode/python-sscha#160] [Q&A] [answered]
Dear Developers,

I am currently study the CDW in a system similar to VSe2, where there are two CDW competing orders. I can do the anharmonic calculations of the parent structure to get its free energy (by converging the SSCHA matrix). I am wondering if I can also do the same calculations to its two daughter CDW phases? Usually the daughter CDW phase has a larger primitive cell and can I just use 1 by 1 by 1 to calculate all three phases to get free energy or I need to use supercell? The idea is to see the free energy evolutions of three phases with tempearture.
 
Many thanks for your help.

Best wishes,
Kang

**Comment by ionerrea** [comment, marked as answer]
Dear Kang,

I would try to be consistent with the number of atoms in total. If in the parent phase you need to converge a supercell of a given size, in the daughter CDW phases I would use a supercell with the same number of atoms in total. As the primitive cell is going to be bigger the supercell size will be smaller in the CDW phase.

Is it clear?

Note that even if the number of atoms is the same in all phases, symmetries will be reduced in the CDW phase and you may need a larger number of configurations.

Best,

**Reply by kw600** [reply]
Dear ION,

Many thanks for your reply. I understand it now.

Best wishes,
Kang
